packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -1056,7 +1056,7 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
      * @param node the tree node if available.
      * @param event the mouse single-click event.
      */
-    protected handleClickEvent(node: TreeNode | undefined, event: React.MouseEvent<HTMLElement>): void {
+    protected handleClickEvent(node: TreeNode | undefined, event: MouseEvent | React.MouseEvent<HTMLElement>): void {
```

Comment:
`MouseEvent | ` it does not look correct, we only pass `React.MouseEvent` here, could you revert such changes and look for other fix?